backend/alembic/versions/0032_longer_fs_fields.py

```diff
@@ -0,0 +1,119 @@
+"""empty message
+
+Revision ID: 0032_longer_fs_fields
+Revises: 0031_datetime_to_timestamp
+Create Date: 2025-01-24 02:18:30.069263
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "0032_longer_fs_fields"
+down_revision = "0031_datetime_to_timestamp"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("platforms", schema=None) as batch_op:
+        batch_op.alter_column(
+            "slug",
+            existing_type=mysql.VARCHAR(length=50),
+            type_=sa.String(length=100),
+            existing_nullable=False,
+        )
+        batch_op.alter_column(
+            "fs_slug",
+            existing_type=mysql.VARCHAR(length=50),
+            type_=sa.String(length=100),
+            existing_nullable=False,
+        )
+        batch_op.alter_column(
+            "category",
+            existing_type=mysql.VARCHAR(length=50),
+            type_=sa.String(length=100),
+            existing_nullable=True,
+        )
+
+    with op.batch_alter_table("roms", schema=None) as batch_op:
+        batch_op.alter_column(
+            "file_name", existing_type=mysql.VARCHAR(length=450), nullable=False
+        )
+        batch_op.alter_column(
+            "file_name_no_tags", existing_type=mysql.VARCHAR(length=450), nullable=False
+        )
+        batch_op.alter_column(
+            "file_name_no_ext", existing_type=mysql.VARCHAR(length=450), nullable=False
+        )
+        batch_op.alter_column(
+            "file_extension", existing_type=mysql.VARCHAR(length=100), nullable=False
+        )
+        batch_op.alter_column(
+            "file_path", existing_type=mysql.VARCHAR(length=1000), nullable=False
+        )
+
+    with op.batch_alter_table("users", schema=None) as batch_op:
+        batch_op.alter_column(
```

Comment:
I think this is a leftover from a previous change, we could remove it from this PR.